requirements/prod.txt

```diff
@@ -107,15 +107,15 @@ bitarray == 1.2.1 \
     --hash=sha256:2ed675f460bb0d3d66fd8042a6f1f0d36cf213e52e72a745283ddb245da7b9cf
 bleach==3.1.1 \
     --hash=sha256:aa8b870d0f46965bac2c073a93444636b0e1ca74e9777e34f03dd494b8a59d48
-boto3==1.12.11 \
-    --hash=sha256:41cb779ff393a9a0a3e9dccce7689ec10c383c18deb5c69059181db8c0ccecc3 \
-    --hash=sha256:b21d781a7fb0bfa50771362c3495e30dc3cddd7e91d65c21bcb9fe37acf36b1e
+boto3==1.12.12 \
+    --hash=sha256:08886cd451c2ab89912372262208f2c041f4f88b333291e364723d46e97be5b5 \
+    --hash=sha256:f191e02140e9691e91fac8cb87d74ad111015a1c98c3b99a24bba9982dc5ae7e
+botocore==1.15.12 \
+    --hash=sha256:c4efa61c90604913ac093fe4fe9f47e404ad980bc658a9f5fc1c9046e9fe094d \
+    --hash=sha256:f12dd27c759992460b8ce70bfeed600437829b0293e6a08211237f11757678e5
```

Comment:
😬 reminder - you have to keep these two requirements apart or you'll get a merge conflict **every** day when pyup creates a pr for both boto3 and botocore

Reply:
Weird hack: maybe we should have a separate boto3.txt and botocore.txt file imported from `prod.txt` ?

Reply:
well, we could do, but it's fine as along as there's another requirement separating them by a few lines (e.g. celery)

Reply:
We should really add a comment stating exactly that. I keep forgetting that :disappointed:

Reply:
https://github.com/mozilla/addons-server/pull/13636